### ISSUE TITLE
Is this feature supported on Azure Stack

### ISSUE THREAD

**Issue by liykfrank** [Azure/azure-container-networking#469]
**Is Azure CNI supported on Azure stack?**:

---We installed K8s cluster on Azure Stack via Azure kubernetes engine. I would like to understand if this Azure CNI is supported on Azure stack? If not, is this on your roadmap?

Thanks a million.

**Comment by aanandr** [comment]
@rjaini - can you provide an update on Azure CNI in Azure Stack? I have also assigned this to you.

**Comment by rjaini** [comment]
Currently Azure CNI is not supported but we have plans to support it. I don't have exact timeline for now to communicate.But we have road map to support it.
Adding @walterov as FYI.

**Comment by liykfrank** [comment]
Thank you @rjaini @aanandr .
Since we are building  a hybrid solution.
---On Azure we setup AKS cluster with Azure CNI.
---On Azure Stack, it's quite different, we installed K8S with  Azure Kubernetes-Engine.

Due to Azure CNI is not supported for now, for network we selected kubenet.
 But the 2 way connection between POD and VM was not supported by kubenet. This will be a issue when we deploy Consul in Kubernetes and we want to let the VM nodes out of the Kubernetes cluster can talk with the Consul cluster.
 For a workaround we enabled host-network for the Consul Pods. But host-network is not suggested as a good practice. 
So we are looking for the CNI solution, if this Azure CNI could be working on Azure Stack very soon it will be prefect. Otherwise we may need to consider to select some other solutions such as calico,flannel... 
It will be great if we can have this feature on Azure Stack!
